docs/src/main/sphinx/sql/alter-table.rst

```diff
@@ -82,10 +95,14 @@ Allow everyone with role public to drop and alter table ``people``::
 
     ALTER TABLE people SET AUTHORIZATION ROLE PUBLIC
 
-Set table properties (``x=y``) to table ``users``::
+Set table properties (``x = y``) to table ``people``::
 
     ALTER TABLE people SET PROPERTIES x = 'y'
 
+Set table property ``x`` to its default value in table ``people``::
+
+    ALTER TABLE people SET PROPERTIES x = DEFAULT
+
```

Comment:
Please do add also examples on how to deal with multiple properties:
```
ALTER TABLE a SET PROPERTIES foo=123, bar=456
```

Also do take into consideration  mentioning property names/values containing spaces.

```
ALTER TABLE a SET PROPERTIES " s p a c e "='bar'
```